spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,51 @@
+# rubocop:disable Metrics/BlockLength
```

Comment:
[ ] As mentioned by the previous reviewer, kindly consider fixing the linter error instead of disabling the check, the linter helps us write quality code.